### PR TITLE
Add initial implementation

### PR DESCRIPTION
**Pull request by KevinGrandon**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  "version": "0.0.4",
+  "main": "index.js",
+  "license": "MIT",
+  "files": ["templates", "bin"],
+  "bin": {
+    "create-fusion-plugin": "bin/cli.js"
+  },
+  "dependencies": {
+    "chalk": "^2.4.1",
+    "fusion-scaffolder": "https://github.com/kevingrandon/fusion-scaffolder"
```
Comment (rtsao) [inline package.json]: I think this is in fusionjs org now?

File: templates/plugin/content/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# fusion-plugin-created
+
+You have created a Fusion.js universakl plugin.
+
+* Run linters: yarn lint
+* Run tests: yarn test
+* Build the plugin: yarn transpile
```
Comment (rtsao) [inline templates/plugin/content/README.md]: `yarn prepare`?

File: templates/plugin/content/package.json [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    "fusion-core": "^1.0.0",
+    "fusion-tokens": "^1.0.1",
+    "react": "14.x - 16.x",
+    "react-redux": ">=5.0.6",
+    "redux": ">=3.7.2"
+  },
+  "scripts": {
+    "clean": "rm -rf dist",
+    "lint": "eslint . --ignore-path .gitignore",
+    "transpile": "npm run clean && cup build",
```
Comment (rtsao) [inline templates/plugin/content/package.json]: What about `yarn prepare`? I think that is the standard replacement for "prepublish" so maybe using that makes sense.

File: templates/plugin/content/package.json [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    "build-test": "rm -rf dist-tests && cup build-tests",
+    "just-test":
+      "node_modules/.bin/unitest --browser=dist-tests/browser.js --node=dist-tests/node.js",
+    "test": "npm run build-test && npm run just-test",
```
Comment (rtsao) [inline templates/plugin/content/package.json]: Should `yarn` be used  for these scripts?

File: templates/plugin/content/LICENSE [hunk 5]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2017 Uber Technologies, Inc.
```
Comment (rtsao) [inline templates/plugin/content/LICENSE]: Is right? Maybe for now we can just not include any license by default?

File: package.json (hunk 1 above)
Comment (KevinGrandon) [reply package.json]: This is just a fork until https://github.com/fusionjs/fusion-scaffolder/pull/30 lands and is published with updates. Happy to change this if we land/publish that first.

File: templates/plugin/content/LICENSE (hunk 5 above)
Comment (KevinGrandon) [reply templates/plugin/content/LICENSE]: Yeah, let's remove for now.

File: templates/plugin/content/package.json (hunk 4 above)
Comment (KevinGrandon) [reply templates/plugin/content/package.json]: Will update (just copied from one of our other plugins)

**Review by rtsao — DISMISSED** [review]

**Pull request merged**